app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -509,6 +509,7 @@ pre.onebox code {
   .emoji {
     max-height: 15px;
     margin: 0.2em;
+    min-width: 15px;
```

Comment:
Minor, but is there any reason why we don't set width and height here?

Reply:
For some reason, I thought the width and height were automatically set on the img tag, but that's not the case. Not all emojis have the same size, so setting fixed width and height make them look weird.

Reply:
Hmm, interesting. We set width/height for emojis generally: 

https://github.com/discourse/discourse/blob/58f82ad64f812b8171233053659fc1a8a33c855f/app/assets/stylesheets/common/base/emoji.scss#L1-L5 

Do the same emojis look weird when inserted outside a onebox?

Reply:
No, they don't. I think it's because the emoji parent uses `display: inline-block`.

Reply:
Ah ok. LGTM then!